src/app/containers/Figure/index.jsx

```diff
@@ -17,11 +18,27 @@ const FigureContainer = ({
   captionBlock,
   ratio,
   src,
+  height,
   width,
 }) => (
   <Figure>
     <ImagePlaceholder ratio={ratio}>
-      <Image alt={alt} src={src} width={width} />
+      <PlatformContextConsumer>
+        {platform =>
+          platform === 'amp' ? (
+            <AmpImg
+              alt={alt}
+              attribution={copyright || ''}
```

Comment:
Hmm so this means we'll [be adding an amp attribution](http://reff.it/nrlSBJ)  _and_ rendering the copyright below it. How is `attribution` actually used? Is it meta information or is it read out to screen readers?

Reply:
This doesn't seem to be specified by the AMP-img documentation, but I think